packages/form-builder/addon/components/cfb-form-editor/question-usage.hbs

```diff
@@ -0,0 +1,56 @@
+<UkButton
+  {{on "click" (fn (mut this.modalVisible) true)}}
+  data-test-show-question-usage-modal-btn
+>
+  {{t "caluma.form-builder.question.usage.button"}}
+</UkButton>
+
+<UkModal
+  @visible={{this.modalVisible}}
+  @stack={{true}}
+  @onHide={{fn (mut this.modalVisible) false}}
+  data-test-question-usage-modal
+  as |modal|
+>
+  <modal.header>
+    {{t "caluma.form-builder.question.usage.title"}}
+  </modal.header>
+  <modal.body>
+    {{#if this.forms.isLoading}}
+      <div class="uk-text-center">
+        <UkSpinner @ratio={{1}} />
+      </div>
+    {{else}}
+      <ul class="uk-list uk-list-divider" id="question-list">
```

Comment:
What's the ID for?

Reply:
I think it was a copy-paste-relict. Couldn't find any usage for it.